css/index.css

```diff
@@ -0,0 +1,66 @@
+	header {
+	  padding: 10px;
+	 }
+	.navbar {
+	  padding: 0px;
+	}
+	@media only screen and (max-width: 500px) {
```

Comment:
Chỉ chia thành 2 khoảng :
 @media screen and (max-width: 768px) {
}
@media screen and (max-width: 414px) {
}